crowdin_api/api_resources/security_logs/resource.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+
+from crowdin_api.api_resources.abstract.resources import BaseResource
+from crowdin_api.api_resources.security_logs.enums import SecurityLogEvent
+
+
+class BaseSecurityLogsResource(BaseResource):
+    def get_user_security_logs_path(
+        self, userId: int, securityLogId: Optional[int] = None
+    ):
+        if securityLogId is not None:
+            return f"/users/{userId}/security-logs/{securityLogId}"
+        return f"/users/{userId}/security-logs"
+
+    def list_user_security_logs(
+        self,
+        userId: int,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
+        page: Optional[int] = None,
+        event: Optional[SecurityLogEvent] = None,
+        ipAddress: Optional[str] = None,
```

Comment:
We've just recently enhanced the parameters in the list-methods. Please take a look and update here as well